presto-docs/src/main/sphinx/release/release-0.233.rst

```diff
@@ -0,0 +1,76 @@
+=============
+Release 0.233
+=============
+
+General Changes
+_______________
+* Fix an optimizer failure introduced in ``0.229``, where a ``LIKE`` pattern is deduced into a constant, e.g., ``col LIKE 'a' and col = 'b'``.
+* Fix ``ArbitraryOutputBuffer`` to avoid skewing output data distribution.
+* Fix an issue where :func:`classification_fall_out` cannot be found.
+* Add configuration property ``exchange.async-page-transport-timeout`` to set for server side timeout.
+* Add configuration property ``task.http-timeout-concurrency`` to improve performance of task timeout executors.
```

Comment:
* Add support for async page transport with non-blocking IO. This can be enabled with the ``exchange.async-page-transport-enabled`` configuration property.  Server side page transport timeout is controlled with the ``exchange.async-page-transport-timeout`` configuration property.
* Add support to handle timeouts of http requests using multiple thread pools. This can be controlled with the ``task.http-timeout-concurrency`` configuration property.